src/components/organisms/stake-claim-form.tsx

```diff
@@ -21,23 +23,36 @@ export type ClaimFormValues = z.infer<typeof schema>
 export const ClaimForm = (props: {
   defaultValues: ClaimFormValues
   formId: string
-  maxAmount: string
   onSubmit: (data: ClaimFormValues) => void
 }) => {
+  const { stakeDenom } = useSnapshot(Chain)
   const form = useZodForm({
     schema,
-    defaultValues: props.defaultValues,
+    defaultValues: {
+      ...props.defaultValues,
+      denom: stakeDenom,
+    },
   })
-  const { fee } = useSnapshot(Chain)
+
   return (
     <Form form={form} onSubmit={props.onSubmit} id={props.formId}>
-      <AmountField
-        required
-        name="amount"
-        label="Amount"
-        maxValue={props.maxAmount}
-        denom={getFeeDenom(fee)}
-      />
+      <Grid alignItems="end" gridTemplateColumns={'1fr 150px'} gap={2}>
+        <GridItem>
+          <AmountField
+            required
+            name="amount"
+            label="Amount"
+            balances={[]} // TODO(#79): use claimable amount from validator
+          />
+        </GridItem>
+        <GridItem>
+          <DenomField
+            required
+            name="denom"
+            balances={[]} // TODO(#79): use claimable amount from validator
```

Comment:
Just noting here that this is not needed and will be removed in #86. Either pull request can be merged first.